src/homework/day8/collections/MyBirds.java

```diff
@@ -0,0 +1,52 @@
+package homework.day8.collections;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class MyBirds {
+
+
+    public static int countNumberOfVowels(String word) {
+
```

Comment:
Extra new line. Remove. The same for such cases.

Reply:
Done